src/renderer/pages/Operations/common/utils.ts

```diff
@@ -48,6 +44,12 @@ const TransactionIcons: Record<TransactionType, IconNames> = {
   [TransactionType.MULTISIG_AS_MULTI]: 'transferMst',
   [TransactionType.MULTISIG_APPROVE_AS_MULTI]: 'transferMst',
   [TransactionType.MULTISIG_CANCEL_AS_MULTI]: 'transferMst',
+  // XCM
+  [TransactionType.XCM_LIMITED_TRANSFER]: 'unknownMst',
```

Comment:
Is correct icon still in progress?

Reply:
Yeah, it's going to appear in another PR